client/my-sites/sidebar/index.jsx

```diff
@@ -810,12 +809,16 @@ export class MySitesSidebar extends Component {
 		if ( this.props.isJetpack && ! this.props.isAtomicSite && ! this.props.isVip ) {
 			const urlParts = getUrlParts( site.options.admin_url + 'admin.php' );
 			delete urlParts.search;
-			adminUrl = getUrlFromParts( {
-				...urlParts,
-				protocol: urlParts.protocol || 'https:',
-				searchParams: new URLSearchParams( { page: 'jetpack' } ),
-				hash: '/my-plan',
-			} ).href;
+			try {
+				adminUrl = getUrlFromParts( {
+					...urlParts,
+					protocol: urlParts.protocol || 'https:',
+					searchParams: new URLSearchParams( { page: 'jetpack' } ),
+					hash: '/my-plan',
+				} ).href;
+			} catch ( error ) {
+				return null;
```

Comment:
Is `null` a valid value to return in this context? If so, looks good 👍

Reply:
It should be; components can return `null` to render nothing, and I think that holds true for functions that act like components, like this function. In any case, there was already a `null` return a few lines above so it's at least as safe as that guard.